packages/cross_file/test/x_file_html_test.dart

```diff
@@ -102,17 +106,24 @@ void main() {
 
         final html.Element container =
             html.querySelector('#$crossFileDomElementId')!;
-        final html.AnchorElement element = container.children
-                .firstWhere((html.Element element) => element.tagName == 'A')
-            as html.AnchorElement;
+
+        html.HTMLAnchorElement? element;
+        for (int i = 0; i < container.childNodes.length; i++) {
```

Comment:
I'm hoping a lot of this replicating-list-behavior business goes away when we can start treating some interop types as iterable.

Reply:
yes please!